crs_support/urdf/crs.urdf.xacro

```diff
@@ -343,7 +343,7 @@
         </collision>
         <collision> Back Wall bottom Half
             <geometry>
-                <box size="0.1 1.1 3.0" />
+                <box size="0.4 1.1 3.0" />
```

Comment:
What changed in the workcell that led to this change?

Reply:
This is to account for the tool change stations, I just made that wall thicker